lib/util.py

```diff
@@ -122,3 +123,29 @@ def shell_command(command):
     else:
         return out, err
 
+# ## Many of the view functions just do a print without any valid returns
+# ## This is to capture such prints so that they can be used as returns
+def capture_stdout(func,line=[''],printMsg=True):
+    name = ''
+    if printMsg is True:
+        try:
+            name = func.func_name
+        except Exception:
+            pass
+        print "Data collection for " + name +str(line) + " in progress.."
```

Comment:
done, changed printMsg to collectinfo_msg and moved the other print within the if check block.